docs/layouts/grid-layout.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebarDepth: 0
+---
+
+# GridLayout
+
+This is an overview of the most common usage of GridLayout.
+`GridLayout` is a layout container that lets you arrange its child in a table-like manner.
+
```

Comment:
https://nativescript-vue.org/en/docs/elements/layouts/grid-layout/
has more explanation, you may enhance descriptions here